event-svc/src/store/metrics.rs

```diff
@@ -123,9 +123,9 @@ impl<S: Send + Sync> StoreMetricsMiddleware<S> {
 }
 
 #[async_trait]
-impl<S> ceramic_api::InterestStore for StoreMetricsMiddleware<S>
+impl<S> ceramic_api::InterestService for StoreMetricsMiddleware<S>
```

Comment:
This is still needed in the Event Service?

Reply:
nope good catch will remove, and the mirror as well.